modman

```diff
@@ -1,2 +0,0 @@
-app/code/community/Meanbee/Footerjs/ app/code/community/Meanbee/Footerjs/
```

Comment:
I agree. Got the feedback from colleagues in the past weeks, mentioning that without modman there was no issue. After reusing it myself today, it looks like it was related to an issue mentioned earlier today by Andrew Howden on StackExchange for Magento 2 (also applies here): https://magento.stackexchange.com/questions/177924/magenta-2-1-5-composer-install-fails-to-generate-all-files. I'll put it back where it belongs.